test/SILOptimizer/definite-init-wrong-scope2.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-swift-frontend -emit-sil %s -Onone -Xllvm -sil-print-debuginfo \
+// RUN:   -o /dev/null -Xllvm -sil-print-after=definite-init \
+// RUN:   -Xllvm -sil-print-only-functions=$S8patatino4BearC6before7before26during5afterACSb_S3btKcfc \
```

Comment:
Unless it makes the CHECK lines much messier, it might be better not to filter away SIL by mangled name (w.r.t avoiding test updates if the mangling changes).

Reply:
(Otherwise this lgtm!)

Reply:
Makes sense, let me update and test again :)